manifest.yml.j2

```diff
@@ -57,6 +57,21 @@
       'production': 1
     },
   },
+  'notify-delivery-worker-ecs-fixup': {
+    'disk_quota': '10G',
+    'memory': '4G',
+    'additional_env_vars': {
+      'NOTIFICATION_QUEUE_PREFIX': NOTIFICATION_QUEUE_PREFIX + '-ecs',
+      'CELERYD_PREFETCH_MULTIPLIER': 1,
```

Comment:
No action needed:

This will make our worker a bit slower... but it will mean that if it has to pull down a bunch of reporting tasks (usually run by the `reporting-worker` then it will process them well (because those tasks can take longer than 5 minutes to run and the prefetch_multiplier reduces the chance of duplicates).

I think this is worth it